buildfile

```diff
@@ -1,4 +1,4 @@
-./: {*/ -build/ -upstream/} doc{UPSTREAM_README.md} legal{LICENSE} manifest
+./: {src/ -build/ -upstream/} doc{UPSTREAM_README.md} legal{LICENSE} manifest
```

Comment:
It’s best and idiomatic to include all directories and exclude the ones not need. Here `*/` or `src/` should not make a difference